DESCRIPTION

```diff
@@ -3,7 +3,8 @@ Type: Package
 Title: SDTM Data Transformation Engine
```

Comment:
Do we need to open the abbreviation here too?

Reply:
I think we might be ok. admiral has ADaM in the title.